server/hazard-service/src/main/java/edu/illinois/ncsa/incore/service/hazard/controllers/TornadoController.java

```diff
@@ -264,8 +263,15 @@ public Tornado createTornado(
                 // Store the dataset
                 datasetId = ServiceUtil.createDataset(datasetObject, this.username, this.userGroups, files);
                 // Assuming only one hazardDataset will be created here.
-                TornadoHazardDataset hazardDataset = tornadoModel.getHazardDatasets().get(0);
+                // construct tornado hazardDatasets
```

Comment:
With this logic, I assume we will never have a case where multiple tornado files would be uploaded? or we are keeping this approach so that the way we create tornado doesn't change? I thought of doing this but then I assumed we needed to update how we create tornadoes altogether... I think this looks good!